pstress/pstress-run.conf

```diff
@@ -70,6 +70,22 @@ WORKDIR=$HOME/test_p/$RANDOMD
 ################################################################################
 RUNDIR=/tmp/$RANDOMD
 
+################################################################################
+# The option is used to stop the server in different ways                      #
+# for ex -                                                                     #
+# SIGNAL=RANDOM   => This will randomly stop the server with SIG9, SIG4,       #
+#                    or a normal shutdown                                      #
+# SIGNAL=9        => This will kill the server with SIG9 only                  #
+# SIGNAL=4        => This will kill the server with SIG4 only                  #
+# SIGNAL=SHUTDOWN => This will stop the server normally                        #
+#                                                                              #
+# Note : Any other value apart from these is invalid                           #
+# Default : SIGNAL=RANDOM                                                      #
+# If the purpose of the runs is to perform crash recovery testing, please      #
+# set SIGNAL=9 or SIGNAL=4                                                     #
+################################################################################
+SIGNAL=RANDOM
```

Comment:
the default value of the signal should be 9 or 4. Do not mix them both. They don't serve any purpose together. Make it default as 9.
 signal 4 should be only for special purposes when passed by the user.  signal 9 is more close to true crash recovery. 
Also, use innodb_fast_shutdown=0,1,2 . It is a global variable to control if you want an instant shutdown.

Reply:
I am planning to make following changes

The option will accept only 3 values 0,4,9
The values can be passed comma seperated for ex - SIGNAL=0,4,9 (A user must know what he/she is enabling)
If value is set to SIGNAL=4, it will use kill -4 only
If value is set to SIGNAL=9, it will use kill -9 only
If value is set to SIGNAL=0, it will use normal shutdown (i.e innodb_fast_shutdown=1)
Default value would be 9
innodb_fast_shutdown=0 is for slow shutdown and it is advised to be used during upgrades
innodb_fast_shutdown=2 is similar to kill -9
Hence, sticking with the default_value of 1.